src/server/api/tests/drb.test.ts

```diff
@@ -0,0 +1,95 @@
+import { fetchDRBResults } from "../drb"
+import type { DRBResults } from "../../../types/drbTypes"
+
+jest.mock("../../nyplApiClient", () => {
+  return jest
+    .fn()
+    .mockImplementationOnce(async () => {
+      return await new Promise((resolve) => {
+        resolve({
+          get: jest.fn().mockReturnValueOnce({
+            data: {
```

Comment:
can this object be imported from a separate file?

Reply:
Jest mocks don't seem to allow for it unfortunately.